vision/airdrop_vision_pipeline.py

```diff
@@ -0,0 +1,148 @@
+        time.sleep(0.1)
+
+        # Check if all images have been taken
+        all_images_taken = pipe_utils.flyover_finished(state_path)
+
+        # Load in the json containing the camera data
+        image_parameters: dict[str, consts.CameraParameters] = pipe_utils.read_parameter_json(
+            camera_data_path
+        )
+
+        # Loop through all images in the json - if it hasn't been processed, process it
+        for image_path in image_parameters.keys():
+            if image_path not in completed_images:
+                # Save the image path as completed so it isn't processed again
+                completed_images.append(image_path)
+
+                # Load the image to process
+                image: consts.Image = cv2.imread(image_path)
+
+                # Get the camera parameters from the loaded parameter file
+                camera_parameters: consts.CameraParameters = image_parameters[image_path]
+
+                # Find potential judges in the image
+                current_data = emg_obj.find_humanoids(
+                    image, emg_model, camera_parameters, image_path
+                )
+
+                if not first_detection:
+                    judges: list[BoundingBox] = compare_data(prev_data, current_data)
+
+                    judge_dict: consts.ODLC_Dict = create_judge_dict(judges)
+
+                    pipe_utils.output_odlc_json(output_path, judge_dict)
+
+                prev_data = current_data
+
+                first_detection = False
+
+
+def compare_data(
+    prev_data: list[BoundingBox], current_data: list[BoundingBox]
+) -> list[BoundingBox]:
+    """ "
+    Compares humanoid detections between two images to filter out false positive detections
+
+    Parameters
+    ----------
+    prev_data: list[BoundingBox]
+        The detected humanoids from the previous image being considered
+    current_data: list[BoundingBox]
+        The detected humanoids from the current image
+
+    Returns
+    -------
+    judges: list[BoundingBox]
+        The subset of current_data that is likely to be judges
+    """
+
+    judges: list[BoundingBox] = []
+
+    for current_detection in current_data:
+        min_distance = calc_emerg_obj_min_dist(prev_data, current_detection)
+
+        # Ensures that a detection in the current image that was not in the previous image
+        #   will be ignored
+        if min_distance < MAX_JUDGE_MOVEMENT_F:
+            judges.append(current_detection)
+
+    return judges
+
+
+def create_judge_dict(judges: list[BoundingBox]) -> consts.ODLC_Dict:
+    """
+    Creates an ODLC_Dict object that represents where the judges probably are
+
+    Parameters
+    ----------
+    judges: list[BoundingBox]
+        The list of bounding boxes of judges
+
+    Returns
+    -------
+    judge_dict: ODLC_Dict
+        The dictionary of locations of judges.
+        The keys are indices in string form and are meaningless.
+    """
+
+    judge_dict: consts.ODLC_Dict = {}
```

Comment:
This was kind of an oversight from one of my previous PR reviews, but the `ODLC_Dict` type should be changed to pascal case, rather than uppercase snake case. (no underscore)